types/lodash/lodash-tests.ts

```diff
@@ -6068,20 +6156,45 @@ namespace TestMap {
     let array: number[] | null | undefined = [] as any;
     let list: _.List<number> | null | undefined = [] as any;
     let obj: any = {};
-    let dictionary: _.Dictionary<number> | null | undefined = obj;
+    let dictionary: _.Dictionary<number> | null | undefined = anything;
```

Comment:
Use `const` instead of `let`.

Don't bother updating *all* usages of `let`, otherwise this PR will be nearly impossible to review. But if you're changing a line anyways, might as well change it to `const`.